mock-relying-party-ui-esim/README.md

```
@@ -0,0 +1,68 @@
+
+1. **Home Page**: This page represents the login screen for the relying party's website. This page includes a button with the text, "Sign in with MOSIP". On the click this button, the user gets redirected to the MOSIP's Esignet Portal. The user now has to authenticate and provide consent to share information from MOSIP to relying party, on the Esignet portal.
+
+2. **User Profile Page**: This page shows the user profile on the relying party's website. On successful authentication and consent approval, the user gets navigated to this page with an Auth Code. This Auth Code would be shared with the relying party's backend service via the `/fetchUserInfo` endpoint. The backend then uses the Auth Code to fetch the access token and user details from MOSIP Esignet services.
+
+## Build & run (for developers)
+
+The application run on PORT=5000 by default.
+
+- Env variables
+
+  - ESIGNET_UI_BASE_URL: MOSIP ESIGNET UI URL (Example:https://esignet.dev.mosip.net/)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)